tests/gc/trace_globals.nim

```diff
@@ -15,8 +21,8 @@ proc f() =
   var b {.global.} = init()
   var c {.global.} = init()
 
-  echo a.len
-  echo b.len
-  echo c.len
+  doAssert a.len == 10000000
+  doAssert b.len == 10000000
+  doAssert c.len == 10000000
```

Comment:
I don't remember exactly but I suspect I could trigger the corruptions only with the `echo`, not with `doAssert`... Besides, my fix works too and makes the CI a tiny bit faster.

Reply:
as you wish; note that I can't reproduce the original bug report, see https://github.com/nim-lang/Nim/issues/17085#issuecomment-805355052

for things like this we should add comments like: `# bug triggered by echo` so that tests don't later get transformed into doAssert

(if you merge https://github.com/nim-lang/Nim/pull/17487, I'll update this PR and add this comment)

Reply:
done, I've put back the echo and added a comment